features/text-wrap-mode.yml

```diff
@@ -0,0 +1,6 @@
+name: text-wrap-mode
```

Comment:
I saw that, but opted to use the name because it's already implemented in a couple browsers with that name, and I'm not finding conversations about actually bikeshedding the name.

Reply:
This is no longer an open question. It was [resolved by the group](https://github.com/w3c/csswg-drafts/issues/9102#issuecomment-1758101036) to stick with `text-wrap-mode`.